build.py

```diff
@@ -1,105 +1,60 @@
 import datetime
 
-def generate_blocklist(header_file, sitelist_file, output_file, format, plus_version: bool, sitelist_file_plus='', filterlist_file_ublock_only=''):
+def generate_blocklist(output_file, format, is_sitelist_plus: bool = False, filterlist_file_ublock_only:str = ''):
   """
   Combines and modifies content of two files into a new file, 
   replacing placeholders, counting entries and adding text characters for proper format support.
   """
+  header_file = "header.txt"
+  sitelist_file = "sitelist.txt"
+  sitelist_file_plus = "sitelist-plus.txt"
   # Get current time in desired format with timezone-aware object
   current_time = datetime.datetime.now(datetime.timezone.utc).strftime("%d %b %Y %H:%M UTC")
 
   # Count entries in filtered file, ignoring lines starting with "!"
   num_entries = 0
   with open(sitelist_file, 'r') as f_filtered:
-    for line in f_filtered:
-      if not line.startswith('!'):
-        num_entries += 1
-  if plus_version == True:
-    with open(sitelist_file_plus, 'r') as f_filtered_plus:
-      for line in f_filtered_plus:
-        if not line.startswith('!'):
-          num_entries += 1
+        num_entries += countUncommentedLines(sitelist_file)
+  if(is_sitelist_plus):
```

Comment:
Maybe adding a space makes it look "better" and more consistent with the rest of the code
```suggestion
  if (is_sitelist_plus):
```